components/common/Buttons.tsx

```diff
@@ -104,3 +107,29 @@ export function EditButton({ href }: { href: string }) {
     </Link>
   );
 }
+
+export const ConfirmButton = ({
+  title,
+  buttonRef,
+  onClick,
+}: {
+  title: string;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+  buttonRef?: React.RefObject<HTMLButtonElement | null>;
```

Comment:
```suggestion
  ref?: React.RefObject<HTMLButtonElement | null>;
```

맥락상 버튼의 ref인건 분명한데 프롭 이름을 바꾸는건 어떤가요?

Reply:
좋습니다! 반영 했어요~